packages/logger/src/Logger.ts

```diff
@@ -1266,6 +1269,16 @@ class Logger extends Utility implements LoggerInterface {
             ).toLowerCase() as keyof Omit<LogFunction, 'critical'>);
       this.console[consoleMethod](item.value);
     }
+    if (buffer.evictedLogCount > 0) {
+      this.printLog(
+        LogLevelThreshold.WARN,
+        this.createAndPopulateLogItem(
+          LogLevelThreshold.WARN,
+          `${buffer.evictedLogCount} were evicted from the buffer and are not shown. Increase buffer size to store more logs in the buffer`,
```

Comment:
I think this message could do with some word smithing. I wanted to convey:
1. Logs have been evicted
2. The quantity of logs that have been evicted. Is it meaningful to know 100 logs were evicted vs 1? 
3. What the user can do to change this (i.e. increase buffer size). This is a bit hard to specify as we haven't added the config option yet

Reply:
It might also be good to check what Python does - I will have a look.

Reply:
I think keeping the exact count makes things a bit too brittle. I would just say "some logs might be missing" and leave it at that. Especially because I don't think in most cases you know how big is a log in practice, especially when dealing with input data (i.e. logging a request payload).

This is something we can add in the future if there's demand, but for now I'd just go with the simple option.